### DISCUSSION TITLE
How to access to root state.

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/pinia#1518] [Help and Questions]
I have two store modules settings and user.
<img width="248" alt="Screenshot 2022-08-04 at 08 32 09" src="https://user-images.githubusercontent.com/107699367/182770432-a0d2bfda-1b8b-4ab2-ad62-31c2d88f1081.png">
Is it possible not to call each directly with `useUserStore` and  `useSettingStore`, maybe, Pinia have something like `useGlobalStore` to get root state.

Hypothetical example:
```
const state = useGlobalStore();

state.settings.setColorMode("dark");
state.user.isAuthenticated;
```
Thanks.

**Comment by ghost** [comment]
Certainly, I can create composable function `useGlobalStore` which will return object with two store instance

```
import { useUserStore } from "@/store/user";
import { useSettingsStore } from "@/store/settings";

export const useGlobalStore = () => {
  const user = useUserStore();
  const settings = useSettingsStore();

  return {
    user,
    settings
  };
};

```

**Comment by posva** [comment]
You could create such a function but that would defeat the purpose of splitting into multiple stores to benefit from code splitting and more.

instead, you could use auto imports with unplugin-autoimport
